ceno_zkvm/src/precompiles/lookup_keccakf.rs

```diff
@@ -772,7 +766,7 @@ where
                             c_aux64[i][j] = state64[j][i] ^ c_aux64[i][j - 1];
                             for k in 0..8 {
                                 lk_multiplicity
-                                    .lookup_xor_byte(state8[j][i][k], c_aux8[i][j - 1][k]);
```

Comment:
Previous "assign_instance" operator order mismatch with circuit definition so also lead to lookup failed